Update the documentation with platforms requirements (#37)

cosmin-malutan opened the pull request:
This is for issue #37, I updated the requirements we need to have on client/slave nodes in order to be able to run tps testruns.
@whimboo can you review this?

cosmin-malutan committed:
Update the documentation with platforms requirements (#37)

whimboo commented on README.md:
```diff
@@ -37,15 +37,24 @@ To add Jenkins slaves to your master you have to create new nodes. You can use
+* OpenSSL, Developer Edition: https://slproweb.com/products/Win32OpenSSL.html
+  * Update environment variables:
+  * `SET LIB=C:\OpenSSL-Win32\lib;C:\OpenSSL-Win32\lib\VC\static;%LIB%`
+  * `SET INCLUDE=C:\OpenSSL-Win32\include;%INCLUDE%`
```
Instead of putting all that into our own readme, can't we link to necessary requirement pages for the different modules in use? Thing is that I don't want to update our pages all the time, dependencies are getting changed.

whimboo commented:
I also miss the information of what has to be installed on OS X.

cosmin-malutan committed:
Link to the readme of the required modules

cosmin-malutan commented:
I created the pr for fxa-python-client module with updated requirements: https://github.com/mozilla/fxa-python-client/pull/18 and here I'll just link to that file.

cosmin-malutan commented:
I updated what was required, I added the requirements documentation within dependencies own packages, and linked it here.
@davehunt can you review this?

davehunt commented on README.md:
```diff
@@ -37,9 +37,8 @@ To add Jenkins slaves to your master you have to create new nodes. You can use
-# Documentation has to be updated regarding requirements for fxa-python-client
-# package, issue #37 https://github.com/mozilla/coversheet/issues/37
+Python needs to be installed and be available in PATH.
+Please make sure you have satisfied the requirements for [fxa-python-client](https://github.com/mozilla/fxa-python-client/blob/master/README.md) package
```
Note that the new lines here will not carry across to the rendered document. If you intended these to be on a separate line then you will need to have two new lines. Also, a minor nit but please end your sentence with a period.

davehunt commented:
I think linking out to fxa-python-client makes sense. r- on the minor nit.

cosmin-malutan committed:
Fixed the nit.

cosmin-malutan commented:
@dave, I fixed the nit.
https://github.com/cosmin-malutan/coversheet/blob/issue%2337/README.md

davehunt commented:
r+ thanks, please go ahead and land it.

cosmin-malutan commented:
Something went wrong with the rebase so I still have two commits here. 
Merged as b0b5a91bbd6d0daa72e9f352f35d328ced7fef38 and f8fd7c8b39b33aedc96eb59395cd6618ad56b7af

davehunt commented:
@cosmin-malutan if you have rebase issues again let me know on IRC and I can either help by talking you through it or I can do the rebase/merge for you.
